documentation/developer-onboarding.md

```diff
@@ -142,3 +142,54 @@ HTTP requests from the ingress controller, using the filter from [ingress contro
 ContainerLogV2
 | where LogEntry contains "cpd-production-cpd-ecf-production-web-80"
 ```
+## Monitoring
+
+The main monitoring tools used are Grafana and Alertmanager. For further reading about Monitoring setup in the cluster [click here](https://github.com/DFE-Digital/teacher-services-cloud/blob/97a19da1aedf604e7778b25ce747fd9a9a61a670/documentation/monitoring.md)
+### Grafana
+Grafana could be accessed via the respective URL's based on the environment of interest. The URLs corresponding to each environment are as below
+
+* Test | https://grafana.test.teacherservices.cloud 
+* Production | https://grafana.teacherservices.cloud/
+
+The default access to the grafana interface is view only, which does not require authentication. In order to be able to make changes for example adding more dashboards and editing existing dashboards, requests will have to be made in the #teacher-services-infra slack channel to obtain admin credentials 
+
+### Exporting a Grafana Dashboard as json
+Grafana allows you to export your dashboard as a JSON file, which can be version controlled and shared with others. This could be achieved by following these steps
+- 	Open your dashboard in Grafana
+-	Click on the "Share" button(icon) in the top left corner
+-	In the "Export" tab, select "Export for sharing externally"
+-	Click "Save to file" to download the JSON file of your dashboard
+
+### Editing or creating new grafana dashboards
+The following steps are required for creating or editing dashboards. Please [click](https://grafana.com/docs/grafana/latest/fundamentals/dashboards-overview/) for more extensive details
+-   Ensure you are logged in as an admin
+-   Identify the purpose of your dashboard. What insights the dashboard will provide and what messages it  conveys
+- 	Plan and Design how the dashboard would look when completed, paying attention to the placement of panels, alignment, spacing, colour 
+    and organisation
+- 	Select the Appropriate Data Sources by identifying the right datasource to visualise in the dashboard (examples of usual datasources 
+    are prometheus, , databases, APIs, log files)
+-	Create Panels for each metric by adding panel for the metric and choosing the right visualisation (for example graph, gauge, table, 
```

Comment:
How can users find the available metrics?

Reply:
added